guides/source/documents.yaml

```diff
@@ -140,6 +140,11 @@
       work_in_progress: true
       url: api_app.html
       description: This guide explains how to effectively use Rails to develop a JSON API application.
+    -
+      name: Active Record and PostgreSQL
+      work_in_progress: true
+      url: active_record_postgresql.html
+      description: This guide covers PostgreSQL specific usage of Active Record.
```

Comment:
Don't you guys think that it is worth even having just those points in the guides. The reader could still get a lot of information from that page and otherwise it may be hard to find out that Rails support those custom types.

Since it would be in Digging Deeper section (which already it means that someone is comfortable with Rails) and marked in progress I think it can still bring value even though it doesn't have much prose there.